safe/impact_functions/earthquake/itb_earthquake_fatality_model/impact_function.py

```diff
@@ -115,7 +115,7 @@ def __init__(self):
                 1: 0, 2: 0, 3: 0, 4: 0, 5: 0, 6: 1.0,
                 7: 1.0, 8: 1.0, 9: 1.0, 10: 1.0
             }),
-            ('mmi_range', range(2, 10)),
+            ('mmi_range', range(4, 11)), # from MMI 4 to 10
```

Comment:
We can specify it with the step parameter. It is now set to 0.5 which will pull +/-0.5 for each MMI e.g for calculating the exposed, displaced, and fatality for MMI 2, it will pull all the MMI in interval (1.5, 2.5].